go.mod

```diff
@@ -22,6 +22,7 @@ require (
 	github.com/xdg-go/stringprep v1.0.2
 	github.com/youmark/pkcs8 v0.0.0-20181117223130-1be2e3e5546d
 	golang.org/x/crypto v0.0.0-20201216223049-8b5274cf687f
+	golang.org/x/net v0.0.0-20220624214902-1bab6f366d9e // indirect
```

Comment:
Optional: Consider adding a comment that mentions we have this indirect dependency because it resolves CVE-2021-44716.